data/instruct_advanced_postgres.csv

```diff
@@ -36,12 +36,12 @@ Customer Rank (CR) is determined by ranking customers based on their total trans
-broker,keywords_ratio,"What is the SPM for each ticker symbol from sell transactions in the past month, inclusive of 1 month ago? Return the ticker symbol and SPM.","SPM (Selling Profit Margin) = (Total Amount from Sells - (Tax + Commission)) / Total Amount from Sells * 100","SELECT sbTickerSymbol, (SUM(sbTxAmount) - SUM(sbTxTax + sbTxCommission)) / SUM(sbTxAmount) * 100 AS SPM FROM sbTransaction JOIN sbTicker ON sbTransaction.sbTxTickerId = sbTicker.sbTickerId WHERE sbTxType = 'sell' AND sbTxDateTime >= NOW() - INTERVAL '1 month' GROUP BY sbTickerSymbol","To calculate the Total Active Customers who have recently joined, the Average Closing Price of tickers within a recent period, and the Customer Rank by their total transaction volume, you would need different queries and calculations that are not directly related to calculating the Net Profit Margin (NPM) for sell transactions of ticker symbols in the past month. ACP = Calculate the Average Closing Price of tickers over a chosen recent time span Analyze customer signups over a certain period.
+broker,keywords_ratio,"What is the SPM for each ticker symbol from sell transactions in the past month, inclusive of 1 month ago? Return the ticker symbol and SPM.","SPM (Selling Profit Margin) = (Total Amount from Sells - (Tax + Commission)) / Total Amount from Sells * 100","SELECT sbTickerSymbol, (SUM(sbTxAmount) - SUM(sbTxTax + sbTxCommission)) / SUM(sbTxAmount) * 100 AS SPM FROM sbTransaction JOIN sbTicker ON sbTransaction.sbTxTickerId = sbTicker.sbTickerId WHERE sbTxType = 'sell' AND sbTxDateTime >= CURRENT_DATE - INTERVAL '1 month' GROUP BY sbTickerSymbol;SELECT t.sbTickerSymbol, (SUM(st.sbTxAmount) - (SUM(st.sbTxTax) + SUM(st.sbTxCommission))) / NULLIF(SUM(st.sbTxAmount), 0) * 100 AS SPM FROM sbTransaction st JOIN sbTicker t ON st.sbTxTickerId = t.sbTickerId WHERE st.sbTxType = 'sell' AND st.sbTxDateTime >= DATE_SUB(CURRENT_DATE, INTERVAL 1 MONTH) GROUP BY t.sbTickerSymbol;","To calculate the Total Active Customers who have recently joined, the Average Closing Price of tickers within a recent period, and the Customer Rank by their total transaction volume, you would need different queries and calculations that are not directly related to calculating the Net Profit Margin (NPM) for sell transactions of ticker symbols in the past month. ACP = Calculate the Average Closing Price of tickers over a chosen recent time span Analyze customer signups over a certain period.
```

Comment:
seems like we're adding a query here that uses `DATE_SUB`, which doesn't seem to be supported in postgresql (only in mysql). Tried running but got the following error:
```sql
broker=# SELECT t.sbTickerSymbol, (SUM(st.sbTxAmount) - (SUM(st.sbTxTax) + SUM(st.sbTxCommission))) / NULLIF(SUM(st.sbTxAmount), 0) * 100 AS SPM FROM sbTransaction st JOIN sbTicker t ON st.sbTxTickerId = t.sbTickerId WHERE st.sbTxType = 'sell' AND st.sbTxDateTime >= DATE_SUB(CURRENT_DATE, INTERVAL 1 MONTH) GROUP BY t.sbTickerSymbol;
ERROR:  syntax error at or near "1"
LINE 1: ...t.sbTxDateTime >= DATE_SUB(CURRENT_DATE, INTERVAL 1 MONTH) G...
```

Reply:
Ah yikes this was a typo! 🤦🏽‍♂️ Thanks for the eagle eyed find!